code/__DEFINES/flags.dm

```diff
@@ -72,6 +72,7 @@
 #define EASYDISMEMBER	20
 #define EASYLIMBATTACHMENT 21
 #define TOXINLOVER		22
+#define DIGITIGRADE		23	//Uses weird leg sprites. Optional for Lizards, required for ashwalkers. Don't give it to other races unless you make sprites for this (see human_parts_greyscale.dmi)

```

Comment:
can this please be renamed to something that isn't only known to furries

not all of us jerk off to animals and if we find a bug with the weird lizard legs we shouldn't have to google fetish porn shit just to figure out what the name of shit we need to fix it